README.md

```diff
@@ -1,47 +1,113 @@
-# Artifact Registry Contract
+# Seasons Smart Contract
 
-This upgradable contract registers the Artizen Seasons and each submission (project) to a season. It also mints open edition Artifacts to each submission and has inherited functionalities from the Open Zeppelin ERC1155Upgradeable,OwnableUpgradeable and ERC1155URIStorageUpgradeable contracts.
+This is a smart contract written in Solidity that implements a Seasons marketplace for minting and trading ERC-1155 tokens. The contract allows users to create submissions for different seasons, mint tokens for these submissions, calculate top submissions for each season, and manage fees and royalties.
 
-### function setTreasuryAddress
+## Prerequisites
 
-- Use this function to set central treasury wallet address after deployment. This wallet will securely keep funds until the end of the season.
```

Comment:
Also give a short intro explaining the test net is Goerli and its id (5)...and how to get Goerli ETH via faucet